release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2347,6 +2347,46 @@ Identify the disk that holds the live image, in this example `nvme0n1p3`, and ru
 
 * There is currently a known issue with OVN-Kubernetes that whenever the `NetworkManager` service restarts the node will lose network connectivity and must be recovered. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2074009[*BZ#2074009*])
 
+* If you have {sandboxed-containers-first}, when upgrading a cluster, you might encounter an issue where the Machine Config Operator (MCO) pod changes to a `CrashLoopBackOff` state and the `openshift.io/scc` annotation of the pod displays `sandboxed-containers-operator-scc` instead of the default `hostmount-anyuid` value.
++
+If this happens, temporarily change the `seLinuxOptions` strategy in the `sandboxed-containers-operator-scc` SCC to the less restrictive `RunAsAny`, so that the admission process does not prefer it over the `hostmount-anyuid` SCC.
++
+. Change the `seLinuxOptions` strategy by running the following command:
++
+[source,terminal]
+----
+$ oc patch scc sandboxed-containers-operator-scc --type=merge --patch '{"seLinuxContext":{"type": "RunAsAny"}}'
+----
+
+. Restart the MCO pod by running the following commands:
++
```

Comment:
@MirzWeiss you could add substeps before each of these code blocks to describe what each of them does. My assumption is that the first one reduces the MCO replicas to 0 and the second one sets it back to 1, but I'm not 100% sure. By explicitly describing each command in substeps, the reader will understand what each command is for.

Please let me know if you would like to add that now or in a separate PR at a later time. Thanks.

Reply:
@pneedle-rh - since the SME for this issue is on PTO until after the release, I'd prefer to add it in a later PR.